pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -216,6 +216,9 @@ def work(self):
                                         if len(pokemon_to_transfer) == 0:
                                             raise RuntimeError(
                                                 'Trying to transfer 0 pokemons!')
+                                        # Add slight delay between capture & candy transfer #774
+                                        logger.log("Waiting briefly before transferring pokemon")
+                                        sleep(3)
```

Comment:
To be fair, the sleep implementation allows for randomness..
https://github.com/PokemonGoF/PokemonGo-Bot/blob/9ea015e1fd9c5d269276de8a8e822ee8eb0afa0f/pokemongo_bot/human_behaviour.py#L8-L11